Formula/m/manim.rb

```diff
@@ -229,6 +229,15 @@ def install
     venv.pip_install_and_link buildpath
   end
 
+  def caveats
+    <<~EOS
+      To render Tex or MathTex, latex must be accessible from your PATH.
+
+      You can install the recommended latex distribution from Homebrew Cask:
+        brew install --cask mactex-no-gui
+    EOS
+  end
```

Comment:
Thanks for the PR @unitof, but I don't think this should be accepted as-is because:

1. [Caveats are meant for Homebrew-specific packaging issues](https://docs.brew.sh/Formula-Cookbook#caveats), while this is a general problem [documented upstream](https://docs.manim.community/en/stable/installation/macos.html#optional-dependencies).
2. Casks are not available on Linux. (We do have `texlive` however.)

Reply:
Makes sense, no worries!